server/handlers.go

```diff
@@ -284,7 +284,7 @@ func (svr *Server) handlePostShared(
 	case common.V1EigenDABackend:
 		certVersion = certs.V0VersionByte
 	case common.V2EigenDABackend:
-		certVersion = certs.V1VersionByte
+		certVersion = certs.V2VersionByte
```

Comment:
add comment here that dispersals only support the latest version.
Is that really what we want though? What if a rollup is using an old certVerifier...?
Shouldnt we query the latest cert version their certverifier (or router) supports and then disperse to that?

Reply:
Oh I might have wrongly extrapolated here, or misremembered our previous conversations. I agree this is fine for this single migration.
But what about when we add a V4 cert for example. Is the idea to deal with that case only then? If so then feel free to resolve this discussion

Reply:
> Oh wait does the payload disperser already set a version? From reading the certVerifier contract? 
Yes there is a version method returned in the EigenDA Cert interface. Added an issue for tracking:
https://github.com/Layr-Labs/eigenda-proxy/issues/404